app/models/setting/provisioning.rb

```diff
@@ -78,6 +81,8 @@ def self.default_settings
       ),
       set('maximum_structured_facts', N_("Maximum amount of keys in structured subtree, statistics stored in foreman::dropped_subtree_facts"), 100, N_('Maximum structured facts')),
       set('default_global_registration_item', N_("Global Registration template"), 'Global Registration', N_("Default Global registration template")),
+      set('ct_command', N_("CoreOS Transpiler Command"), [which('ct'), '--pretty', '--files-dir', transpiler_path], N_("Full patch to CoreOS transpiler (ct) with arguments as an comma-separated array")),
```

Comment:
Would it be better to use the name under which development continues? Flatcar?

Reply:
I am open for discussion but it looks like Flatcar is not the only one in the world and CoreOS (or maybe we can decide something like "Legacy CoreOS") is quite a good term.

https://www.coreos-eol.com/